src/content/test/semantics/headers.tsx

```diff
@@ -21,25 +21,25 @@ export const infoAndExamples = create(({ Markup, Link }) => (
             Add programmatically identified headers to every data table:
             <ol>
                 <li>
-                    If the table has only one row or one column of headers, use <Markup.Term>{`<th>`}</Markup.Term> elements for header
-                    cells and <Markup.Term>{`<td>`}</Markup.Term> elements for data cells.
+                    If the table has only one row or one column of headers, use <Markup.Code>{`<th>`}</Markup.Code> elements for header
```

Comment:
Should the td/th references in here be using `<Markup.Tag tagName="whatever" />`?